### DISCUSSION TITLE
Dark Mode automation?

### DISCUSSION THREAD

**Discussion by scottromack** [tailwindlabs/tailwindcss#4408] [General]
I'm new so this may be a stupid question but, from what I am seeing we are to add a dark variant to every component. It would be great - though difficult I know - to just transform all color variables to make sense in dark mode, or, to be able to set the dark variant on the root element or body?

Thanks for building the anti-framework I wanted to build!

**Comment by samarmohan** [comment]
You could make the background color dark and pick colors that fit nicely. A good example is [DogeHouse](github.com/benawad/dogehouse). They have #0b0e11 as the background (a nice dark) and have #fd4d4d and grays as the accent. There is no `dark` modifier.
